### PR TITLE
Growth stage!

### PR DESCRIPTION
**Pull request by mcollina**
https://github.com/openjs-foundation/cross-project-council/pull/626

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
Can somebody else do the equivalent swap on the website? Thanks!

File: README.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ Great contributors on a specific area in the Fastify ecosystem will be invited t
 
 [<img src="https://github.com/openjs-foundation/cross-project-council/blob/master/logos/openjsf-color.png?raw=true" width="250px;"/>](https://openjsf.org/projects/#incubating)
 
-We are currently an [incubated project](https://openjsf.org/blog/2019/11/20/web-framework-fastify-joins-openjs-foundation-as-an-incubating-project/) at the OpenJS Foundation.
+We are a Growth Project at the OpenJS Foundation.
```
Comment (jsumners) [inline README.md:250]:
```suggestion
We are a Growth Project in the OpenJS Foundation.
```

Or "within". You choose.

**Review by delvedor — COMMENTED** [review]
Should we add a link of what a "growth project" means?

**Review by delvedor — APPROVED** [review]
LGTM 🎉

**Review by fox1t — APPROVED** [review]
LGTM 🎉

**Review by Eomm — APPROVED** [review]

**Pull request merged**